From 1d034a9269fc6e697c5ac20f4ef390a2e8a629d6 Mon Sep 17 00:00:00 2001
From: "Guoniu.Zhou" <guoniu.zhou@nxp.com>
Date: Mon, 8 May 2017 17:47:24 +0800
Subject: [PATCH 1757/5242] MLK-14844 pxp: Add RGB32 format to
 back-compatible.

commit  597f2d9d3e90a7448cbca3a22df7fc9978fd18d6 from
https://source.codeaurora.org/external/imx/linux-imx.git

The latest pxp_dma.h file change PXP_PIX_FMT_RGB32 to PXP_PIX_FMT_XRGB32 format,
but the userspace still use PXP_PIX_FMT_RGB32, so add back it and keep the same
with PXP_PIX_FMT_XRGB32 format.

Signed-off-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/uapi/linux/pxp_dma.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/linux/pxp_dma.h b/include/uapi/linux/pxp_dma.h
index 6a6a12a..4fb2aeb 100644
--- a/include/uapi/linux/pxp_dma.h
+++ b/include/uapi/linux/pxp_dma.h
@@ -66,6 +66,7 @@
 #define PXP_PIX_FMT_ARGB32  fourcc('A', 'R', 'G', 'B')	/*!< 32  ARGB-8-8-8-8 */
 #define PXP_PIX_FMT_RGBA32  fourcc('R', 'G', 'B', 'A')	/*!< 32  RGBA-8-8-8-8 */
 #define PXP_PIX_FMT_ABGR32  fourcc('A', 'B', 'G', 'R')	/*!< 32  ABGR-8-8-8-8 */
+#define PXP_PIX_FMT_RGB32	PXP_PIX_FMT_XRGB32
 /*! @} */
 /*! @name YUV Interleaved Formats */
 /*! @{ */
-- 
1.7.9.5

